pycqed/analysis/measurement_analysis.py

```diff
@@ -3920,36 +4095,36 @@ def run_default_analysis(self, print_fit_results=False,
             Qe = abs(Q / abs(1 - S21min))
 
             # Note: input to the fit function is in GHz for convenience
-            HangerModel.set_param_hint('f0', value=f0*1e-9,
+            Model.set_param_hint('f0', value=f0*1e-9,
```

Comment:
Model value *1e-9 still scares me but I assume this is not solved in this PR.

Reply:
Yes, things like this scare me too, but the reason this is so here and in all my PRs is because the HangerFuncAmplitude and the SlopedHangerFuncAmplitude models were both defined with inconsistent units. I should have changed this but for some reason I didn't get to.

![hangerfuncamplitude](https://user-images.githubusercontent.com/14844492/31719978-855b7132-b415-11e7-8b33-6dbfa69fae93.png)

![slopedhangerfuncamplitude](https://user-images.githubusercontent.com/14844492/31719992-91c40b8c-b415-11e7-975d-930ba7e8af6f.png)